censys/asm/inventory.py

```diff
@@ -31,17 +31,29 @@ def search(
         Returns:
             dict: Inventory search results.
         """
+        if workspaces is not None:
+            print(
+                "The field 'workspaces' is being deprecated. The workspace associated with `CENSYS-API-KEY` will be used automatically."
+            )
         if page_size is None:
             page_size = 50
+        w: List[str] = []
+        w.append(self.get_workspace_id())
```

Comment:
Maybe we could combine these two lines so that we initialize the workspaces variable and get the ID.